server/src/main/java/pl/bk/pizza/store/domain/discount/DiscountPriority.java

```diff
@@ -0,0 +1,16 @@
+package pl.bk.pizza.store.domain.discount;
+
+import com.google.common.collect.ImmutableSortedMap;
+import pl.bk.pizza.store.domain.discount.bonus.ExtraProductDiscount;
+import pl.bk.pizza.store.domain.discount.bonus.PercentDiscount;
+import pl.bk.pizza.store.domain.discount.bonus.TotalPriceDiscount;
+
+public class DiscountPriority
```

Comment:
\[Checkstyle\] ERROR: Utility classes should not have a public or default constructor\.